.gitignore

```diff
@@ -1,2 +1,72 @@
+# ------- Node ignore -------
+
+# Logs
+logs
+*.log
+npm-debug.log*
+
+# Diagnostic reports (https://nodejs.org/api/report.html)
+report.[0-9]*.[0-9]*.[0-9]*.[0-9]*.json
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Dependency directories
 node_modules/
+jspm_packages/
+
+# TypeScript cache
+*.tsbuildinfo
+
+# Optional eslint cache
+.eslintcache
+
+# dotenv environment variable files
```

Comment:
This is an optional trim if the desired outcome is to reduce the amount of files in `.gitignore`. As far as I'm aware, `.env` should be the only file used for environment variables.  Local / production environments usually have their own instance of a `.env` file.